packages/deployments/contracts/src/cli/init/config/mainnet/production.ts

```diff
@@ -338,28 +338,28 @@ export const MAINNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
-    {
-      // TODO: This is the old config, to be removed in execution layer upgrade
-      name: "ALCX",
-      canonical: {
-        domain: "6648936",
-        address: "0xbd18f9be5675a9658335e6b7e79d9d9b394ac043",
-        decimals: 18,
-        cap: utils.parseUnits("10000000", 18).toString(),
-      },
-      representations: {
-        /// ARBITRUM
-        "1634886255": {
-          local: "0x27b58D226fe8f792730a795764945Cf146815AA7",
-          adopted: "0x27b58D226fe8f792730a795764945Cf146815AA7",
-        },
-        /// OPTIMISM
-        "1869640809": {
-          local: "0xE974B9b31dBFf4369b94a1bAB5e228f35ed44125",
-          adopted: "0xE974B9b31dBFf4369b94a1bAB5e228f35ed44125",
-        },
-      },
-    },
+    // {
```

Comment:
do we want to uncomment?

Reply:
No, this actually messes with the init. It's just here so we remember to remove this asset later.